README.md

```diff
@@ -18,32 +16,21 @@ how to customize its behavior for your application.
 
 ## Overview
 
-**Nodogsplash** (NDS) is a high performance, small footprint Captive Portal, offering by default a simple splash page restricted Internet connection, yet incorporates an API that allows the creation of sophisticated authentication applications.
-
-**If you want to provide simple and immediate public access** to an Internet connection with users giving some acknowledgment of the service, Nodogsplash does this by default.
-Customising the page seen by users is a simple matter of editing the simple default html splash page file.
-
-**If you want to enforce use of a set of preset usernames** and passwords with perhaps a limited connection time, the addition of a simple shell script is all that is required.
+**Nodogsplash** (NDS) is a high performance, small footprint Captive Portal, offering a simple splash page restricted Internet connection.
 
-**If you want a more sophisticated authentication system** providing a dynamic web interface you can do that too by providing your own web service written in a language such as php served either by NDS itself or a separate web service.
+NoDogSplash is optimised for target devices with limited resources.
 
-**If you want to link Nodogsplash** to your own centralised Internet based authentication service with user account self generation and access charging, you can do that too, or anything in between.
```

Comment:
"you need the openNDS package rather than NoDogSplash."
=>
"you need [openNDS](https://github.com/opennds) rather than NoDogSplash.

Reply:
There is a link lower down, but yes.

Reply:
ok, I didn't check.